### PR TITLE
Add managed authentication documentation

### PR DESCRIPTION
**Pull request by ThumulaPerera**
## Purpose
$subject

## Related Issue
https://github.com/wso2-enterprise/choreo/issues/25059

### CODE REVIEW

**Review by shaniR — DISMISSED** [review]

**Review by shaniR — DISMISSED** [review]

File: en/docs/authentication-and-authorization/secure-web-applications-with-managed-authentication.md [hunk 1]
```diff
@@ -0,0 +1,214 @@
...
+                });
+
+                const status = refreshResponse.status;
+                if (status === 401) {
+                    // Session has expired (i.e., Refresh token has also expired).
+                    // Redirect to login page
+                    window.location.href = "/auth/login";
+                }
+                if (status !== 204) {
+                    // The STS cannot refresh the token due to a server error.
```
Comment (omindu) [inline en/docs/authentication-and-authorization/secure-web-applications-with-managed-authentication.md:140]: Can we omit the term STS ?

File: en/docs/authentication-and-authorization/secure-web-applications-with-managed-authentication.md [hunk 2]
```diff
@@ -0,0 +1,214 @@
...
+}}>Login</button>`
+```
+
+When a user clicks on the logout button, Choreo will clear the session cookies and redirect the users to the OIDC logout endpoint of the configured Identity Provider (if available).  
+
+### API calls
+
+You can invoke Choreo APIs within the same organization as your web application using the relative path /choreo-apis/<api-suffix>, regardless of whether managed authentication is enabled for the web application or not.
+
+For example, if the API URL is `https://2d9ec1f6-2f04-4127-974f-0a3b20e97af5-dev.e1-us-east-azure.choreoapis.dev/rbln/item-service/api-e04/1.0.0`, the `<api-suffix>` would be `/rbln/item-service/api-e04/1.0.0`. From your Single Page Application, you can invoke the API using `/choreo-apis/rbln/item-service/api-e04/1.0.0` relative path.
```
Comment (omindu) [inline en/docs/authentication-and-authorization/secure-web-applications-with-managed-authentication.md:89]: Should mention this relative URL can be found in web app's dependencies > connection page

File: en/docs/authentication-and-authorization/secure-web-applications-with-managed-authentication.md [hunk 3]
```diff
@@ -0,0 +1,214 @@
...
+| ----------------- | ----------------- | ----------------- |
+| Post Login Path   | This represents the relative path that the application will be redirected to following a successful login. In your code, you will need to implement the necessary logic to handle the userinfo cookie set by Managed Authentication. See [ Handling user info post-login](#handling-user-info-post-login)       | /                      |
+| Post Logout Path  | The relative path to which Choreo redirects after a successful logout.  | /                      |
+| Error Path        | The relative path to which Choreo redirects to if an error occurs during a redirection-based flow (i.e. Login, Logout). See [Custom Error Page](#custom-error-page)             | Builtin error page     |
+| Additional Scopes | All additional scopes required by the web application. The following scopes will be added by default. `openid`, `profile`, `email`, and scopes required to invoke subscribed APIs.               | none                   |
+
+
+## Step 4: Manage OAuth Keys
+
+1. In the left navigation of your component view, click **Settings**.
```
Comment (omindu) [inline en/docs/authentication-and-authorization/secure-web-applications-with-managed-authentication.md:212]: Settings and the to `Authentication Keys` tab

File: en/docs/authentication-and-authorization/secure-web-applications-with-managed-authentication.md [hunk 4]
```diff
@@ -0,0 +1,214 @@
...
+
+| Option            |  Description      | Default           |
+| ----------------- | ----------------- | ----------------- |
+| Post Login Path   | This represents the relative path that the application will be redirected to following a successful login. In your code, you will need to implement the necessary logic to handle the userinfo cookie set by Managed Authentication. See [ Handling user info post-login](#handling-user-info-post-login)       | /                      |
+| Post Logout Path  | The relative path to which Choreo redirects after a successful logout.  | /                      |
+| Error Path        | The relative path to which Choreo redirects to if an error occurs during a redirection-based flow (i.e. Login, Logout). See [Custom Error Page](#custom-error-page)             | Builtin error page     |
+| Additional Scopes | All additional scopes required by the web application. The following scopes will be added by default. `openid`, `profile`, `email`, and scopes required to invoke subscribed APIs.               | none                   |
+
+
+## Step 4: Manage OAuth Keys
```
Comment (omindu) [inline en/docs/authentication-and-authorization/secure-web-applications-with-managed-authentication.md:210]: Can we say we auto-generate keys for web application when in-built IDP is used

File: en/docs/authentication-and-authorization/secure-web-applications-with-managed-authentication.md (hunk 1 above)
Comment (ThumulaPerera) [reply en/docs/authentication-and-authorization/secure-web-applications-with-managed-authentication.md]: Resolved with [c5726a1](https://github.com/wso2/docs-choreo-dev/pull/1183/commits/c5726a15bc558bf96b3fb05ab1f27c4838baf301)

File: en/docs/authentication-and-authorization/secure-web-applications-with-managed-authentication.md (hunk 2 above)
Comment (ThumulaPerera) [reply en/docs/authentication-and-authorization/secure-web-applications-with-managed-authentication.md:89]: Resolved with [c5726a1](https://github.com/wso2/docs-choreo-dev/pull/1183/commits/c5726a15bc558bf96b3fb05ab1f27c4838baf301)

File: en/docs/authentication-and-authorization/secure-web-applications-with-managed-authentication.md (hunk 3 above)
Comment (ThumulaPerera) [reply en/docs/authentication-and-authorization/secure-web-applications-with-managed-authentication.md:218]: Resolved with [c5726a1](https://github.com/wso2/docs-choreo-dev/pull/1183/commits/c5726a15bc558bf96b3fb05ab1f27c4838baf301)

File: en/docs/authentication-and-authorization/secure-web-applications-with-managed-authentication.md (hunk 4 above)
Comment (ThumulaPerera) [reply en/docs/authentication-and-authorization/secure-web-applications-with-managed-authentication.md:213]: Resolved with [c5726a1](https://github.com/wso2/docs-choreo-dev/pull/1183/commits/c5726a15bc558bf96b3fb05ab1f27c4838baf301)

File: en/docs/authentication-and-authorization/secure-web-applications-with-managed-authentication.md [hunk 5]
```diff
@@ -7,6 +7,9 @@ Moreover, Choreo's Managed Authentication introduces the concept of a Backend Fo
 !!! note
     Managed Authentication is currently available only for React, Angular, and Vue.js buildpacks.
 
+!!! warn
+    Managed Authentication relies on "SAMESITE" cookie attribute for security. It is not recommended to be used with older browsers which do not have "SAMESITE" cookie support.
```
Comment (ayomawdb) [inline en/docs/authentication-and-authorization/secure-web-applications-with-managed-authentication.md:11]: - Why do we say so ("Managed Authentication" feature replying on cookies for maintaining user session. Such interactions require CSRF protection. In this context, SAMESITE cookie attribute is used to achieve CSRF protection).
- Are we going to provide instructions on what should be done if developer needs more security (reading non-http-only cookie and sending it in as a parameter, etc.). If CSRF protection is required for older browsers, should they not use "Managed Authentication" at all? 
- Link SAMESITE to https://owasp.org/www-community/SameSite
-

Comment (ayomawdb) [reply en/docs/authentication-and-authorization/secure-web-applications-with-managed-authentication.md:11]: Enables Managed Authentication with CSRF protection using the SAMESITE cookie attribute. Recommended for use with modern browsers that support SAMESITE

Comment (ThumulaPerera) [reply en/docs/authentication-and-authorization/secure-web-applications-with-managed-authentication.md]: Resolved with [79dea23](https://github.com/wso2/docs-choreo-dev/pull/1183/commits/79dea23b581488d180f2f7b43f80c45a33d6b236)

**Pull request merged**